modules/helm-installing-helm.adoc

```diff
@@ -8,17 +8,51 @@
 .Prerequisites
 * You have installed Go, version 1.13 or higher.
 
-.Procedure
-. To install Helm, download the Helm binary and install Helm with the `make` command:
+== On Linux
+
+. Download the Helm binary and add it to your path:
 +
 ----
-$ curl https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest
-$ GO111MODULE=on make
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest/helm-linux-amd64 -o /usr/local/bin/helm
+# chmod +x /usr/local/bin/helm
```

Comment:
Not critical, but just wondering if this deserves to be its own step to change the permissions?

Reply:
Broke it down into two steps.